ws/test_ws_ping.py

```diff
@@ -447,7 +442,6 @@ def test(self):
 
 
 class WsPipelining(WsPing):
```

Comment:
I'd propose to use WssPing as base class since nowadays plain HTTP isn't used and we may expect it only on service requests like PURGEs

Reply:
Done